habitat-lab/habitat/tasks/rearrange/rearrange_task.py

```diff
@@ -82,6 +82,9 @@ def __init__(
         self._episode_id: str = ""
         self._cur_episode_step = 0
         self._should_place_articulated_agent = should_place_articulated_agent
+        # support hab2 benchmark fixed states
+        if self._sim.habitat_config.agents["main_agent"].is_set_start_state:
+            self._should_place_articulated_agent = False
```

Comment:
Note: this has probably been broken for awhile. Without this, all RearrangeTasks were overriding the Hab2 benchmark fixed states, invalidating the benchmark.

Reply:
@ASzot @xavierpuigf check me here on the expected logic.
When this start state is provided in config the `ArticulatedAgentManager.post_obj_load_reconfigure` processes the configured values, so we should respect that and turn off forced re-placement of the agent.

Reply:
Yes agreed.